packages/components/navbar/src/NavbarSwitcher/NavbarEnvVariant.tsx

```diff
@@ -15,11 +19,20 @@ export function NavbarEnvVariant({
   envVariant,
   className,
 }: NavbarEnvVariantProps) {
-  const color = envVariant === 'master' ? tokens.green600 : tokens.gray600;
+  const isMaster = envVariant === 'master';
+  const color = isMaster ? tokens.green700 : tokens.orange700;
 
-  return isAlias ? (
-    <EnvironmentAliasIcon color={color} className={className} size="medium" />
-  ) : (
-    <EnvironmentIcon color={color} className={className} size="medium" />
-  );
+  if (isMaster && !isAlias) {
```

Comment:
Why are we differentiating between master and aliased to master? For the customer it's the same thing, isn't it?

Reply:
It isn't, we change the true master environment icon to  `RocketLaunchIcon`.

Reply:
What I'm saying is, if an alias is pointing to master, it's essentially master.